runner/src/main/java/org/ananas/runner/legacy/core/StepConfig.java

```diff
@@ -55,9 +55,12 @@
 
   // API
   public static final String API_METHOD = "method";
+  public static final String API_FORMAT = "format";
```

Comment:
We should stop using the unified StepConfig class, instead, create a dedicated `APIStepConfig` class in the `steprunner` package, in order to separate concerns for each step.